# GPS MESSAGES
std_msgs/Header header
float64 latitude
string  latitudeRaw
float64 longitude
string  longitudeRaw
float64 altitude
float64 utmEasting
float64 utmNorthing
int32   zone
uint8   zoneLetter
int32   quality
int32   numSatellites
